code/modules/mob/new_player/login.dm

```diff
@@ -17,6 +17,9 @@
 
 	show_lobby_menu()
 
+	if(client && SSinput.initialized)
+		client.set_macros()
```

Comment:
Why isn't this in client/New? What happens if the SS isn't initialized? Feels like it should just go in the prefs thing (if anywhere).

Reply:
I don't think our pref system support this yet.